templates/tower-project.yaml

```diff
@@ -173,7 +173,7 @@ Resources:
               - ec2:DescribeInstanceAttribute
               - ec2:DescribeInstanceTypes
               - ec2:DescribeInstanceStatus
-            Resource: "*"
+            Resource: "*" #Should this be conditional on project resources only?
```

Comment:
Ideally, yes. Unfortunately, as Tess pointed out [here](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/pull/54#discussion_r701339679), we can't because the resources don't exist at the time of deployment. 😔

Reply:
What about if the condition statement is on [resource tags](https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_attribute-based-access-control.html)? The resources don't need to exist, just need be tagged correctly when they are created.